test/Controller/ArticleControllerTest.php

```diff
@@ -5586,16 +5391,14 @@ public function prcVorHistoryProvider()
                 ],
             ],
             [
-                'Version of Record published',
-                'May 3, 2023 (This version)',
-                'Reviewed preprint version 3',
-                'February 18, 2023 (Go to version)',
-                'Reviewed preprint version 2',
-                'February 17, 2023 (Go to version)',
-                'Reviewed preprint version 1',
-                'February 16, 2023 (Go to version)',
-                'Preprint posted',
-                'February 15, 2023 (Go to version)',
+                'Version of Record',
+                'May 3, 2023Read the peer reviews',
```

Comment:
Does this need a space between the date and Read the peer reviews?

Reply:
Thanks @discodavey, It is on purpose, because it is a tricky component, we are building the logic as part of the component and the text "May 3, 2023Read the peer reviews" looks like this:
<dd><time datetime="2024-06-04">June 4, 2024</time><a href="/articles/91452/peer-reviews#content">Read the peer reviews</a></dd>
The test logic is looping through the timeline plain text and that's why there is not space between these two different element. Fixing this will cause a lot of changes.